src/main/java/duke/Times.java

```diff
@@ -0,0 +1,30 @@
+package duke;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+/**
+ * Object class for time
+ * Input of time will be formatted to MMM d yyyy hh:mm
+ */
+public class Times {
+    static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd HHmm");
```

Comment:
If ``FORMATTER`` is not used in other class and try to encapsulate it using ``private``

Reply:
Noted with thanks